apigentools/config.py

```diff
@@ -2,20 +2,20 @@
 # under the 3-clause BSD style license (see LICENSE).
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2019-Present Datadog, Inc.
-import json
+import yaml
+
+from apigentools import constants
 
 
 class Config:
     def __init__(self, raw_dict):
         # TODO: verify the schema of the raw config dict, possibly use jsonschema for that
         self.raw_dict = raw_dict
         self.defaults = {
-            "codegen_exec": "openapi-generator",
```

Comment:
we should probably keep both - codegen_exec and container_image

Reply:
The `codegen_exec` is now basically implied by the `generate` commands used. The default codegen executable is the one used in the default generate command, which is openapi-generator.